mlrun/alerts/alert.py

```diff
@@ -120,10 +122,10 @@ def with_notifications(self, notifications: list[Notification]):
             self.notifications.append(notification)
         return self
 
-    def with_entity(self, entity: alert_constants.EventEntity):
-        if not isinstance(entity, alert_constants.EventEntity):
-            raise ValueError("Entity parameter must be of type: EventEntity")
-        self.entity = entity
+    def with_entity(self, entity: alert_constants.EventEntities):
```

Comment:
```suggestion
    def with_entities(self, entities: alert_constants.EventEntities):
```

If we're renaming, let's adjust the sdk accordingly.